rpc/tendermint/test/common.go

```diff
@@ -32,7 +32,7 @@ func TestWrapper(runner func() int) int {
 func DebugMain() {
 	t := &testing.T{}
 	TestWrapper(func() int {
-		testBroadcastTx(t, "HTTP")
```

Comment:
What happened here?  -- looks like a non-trivial delta out of context, ie I don't get it :)